qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.test_framework import MINIMAL_SC_HEIGHT, MINER_REWARD_POST_H200
```

Comment:
`MINER_REWARD_POST_H200` is unused.
```suggestion
from test_framework.test_framework import MINIMAL_SC_HEIGHT
```

Reply:
Done